### PR TITLE
Add an account dropdown on Teletraan UI to separate it from Display Mode

### PR DESCRIPTION
**Pull request by liyaqin1**

### CODE REVIEW

**Comment by liyaqin1** [comment]
Tested on dev1: https://deploy-dev1.pinadmin.com/env/deploy-open-sentinel-dev1/test-timeout/

File: deploy-board/deploy_board/settings.py [hunk 1]
```diff
@@ -373,4 +373,9 @@
...
 
     #Stage Type Info Link
-    STAGE_TYPE_INFO_LINK = os.getenv("STAGE_TYPE_INFO_LINK")
\ No newline at end of file
+    STAGE_TYPE_INFO_LINK = os.getenv("STAGE_TYPE_INFO_LINK")
+
+    #Primary Account
+    AWS_PRIMARY_ACCOUNT = os.getenv("AWS_PRIMARY_ACCOUNT", "998131032990")
+    #Sub Account
+    AWS_SUB_ACCOUNT = os.getenv("AWS_SUB_ACCOUNT", "562567494283")
```
Comment (ntascii) [inline deploy-board/deploy_board/settings.py:381]: can we support multiple sub accounts?

Comment (liyaqin1) [reply deploy-board/deploy_board/settings.py:381]: yea, this is a very good point. we can support multiple subaccounts in the future. Now, I will add a new option "others" in the dropdown menu.

**Comment by liyaqin1** [comment]
Before:
Screenshot 2023-12-21 at 3 38 41 PM
Now:
Screenshot 2023-12-21 at 3 38 52 PM
Screenshot 2023-12-21 at 3 39 08 PM

**Review by ntascii — APPROVED** [review]
nice

**Pull request merged**